src/main/java/io/weaviate/client6/v1/api/collections/ObjectTtl.java

```diff
@@ -0,0 +1,81 @@
+package io.weaviate.client6.v1.api.collections;
+
+import java.util.function.Function;
+
+import com.google.gson.annotations.SerializedName;
+
+import io.weaviate.client6.v1.api.collections.query.BaseQueryOptions;
+import io.weaviate.client6.v1.internal.ObjectBuilder;
+
+public record ObjectTtl(
+    @SerializedName("enabled") boolean enabled,
+    @SerializedName("defaultTtl") Integer defaultTtlSeconds,
+    @SerializedName("deleteOn") String deleteOn,
+    @SerializedName("filterExpiredObjects") Boolean filterExpiredObjects) {
+
+  public static ObjectTtl of(Function<Builder, ObjectBuilder<ObjectTtl>> fn) {
```

Comment:
nit: Can we use everywhere `ObjectTTL` name?

Reply:
Since v6 we're trying to adhere to Google Java Style Guide, which [insists](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) that only the first letter is capitalized in each "word", e.g. `Xml`, `Http`, `Rest`, etc.

Following this convention, ObjectTTL should be called ObjectTtl.